### PR TITLE
Update geo: test links

### PR DESCRIPTION
**Pull request by rtsisyk**

### CODE REVIEW

File: public/test.html [hunk 1]
```diff
@@ -134,12 +134,16 @@ <h4>Search for something</h4>
 
   <h3><a href="https://en.wikipedia.org/wiki/Geo_URI_scheme">Geo url scheme</a></h3>
   <ol>
-    <li><a href="geo:37.786971,-122.399677">geo:37.786971,-122.399677</a></li>
```
Comment (biodranik) [inline public/test.html:137]: Why did you delete these lines? I'm using them for testing.

File: public/test.html [hunk 2]
```diff
@@ -167,9 +171,7 @@ <h3>Google url scheme</h3>
   <h3>OpenStreetMap url scheme</h3>
   <ol>
     <li><a href="https://www.openstreetmap.org/#map=16/33.89041/35.50664">https://www.openstreetmap.org/#map=16/33.89041/35.50664</a></li>
-    <li><a href="https://openstreetmap.org/#map=16/33.89041/35.50664">https://openstreetmap.org/#map=16/33.89041/35.506641</a></li>
```
Comment (biodranik) [inline public/test.html:170]: Why did you remove this and another test link below? They are useful to test.

Comment (rtsisyk) [reply public/test.html:170]: openstreetmap.org without www is not caught anymore https://github.com/organicmaps/organicmaps/pull/4415

File: public/test.html (hunk 1 above)
Comment (rtsisyk) [reply public/test.html:137]: I just added a comment. This link is just below.

**Review by biodranik — APPROVED** [review]
